ext/QuantumOpticsExt/QuantumOpticsExt.jl

```diff
@@ -70,34 +71,14 @@ express_nolookup(s::XBasisState, ::QuantumOpticsRepr) = (_s₊,_s₋)[s.idx]
 express_nolookup(s::YBasisState, ::QuantumOpticsRepr) = (_i₊,_i₋)[s.idx]
 express_nolookup(s::ZBasisState, ::QuantumOpticsRepr) = (_l0,_l1)[s.idx]
 
-function express_nolookup(o::FockBasisState, r::QuantumOpticsRepr)
-    @warn "Fock space cutoff is not specified so we default to 2"
-    @assert o.idx<2 "without a specified cutoff you can not create states higher than 1 photon"
-    return (_f0₂,_f1₂)[o.idx+1]
-end
-function express_nolookup(o::NumberOp, r::QuantumOpticsRepr)
-    @warn "Fock space cutoff is not specified so we default to 2"
-    return _n₂
-end
-function express_nolookup(o::CreateOp, r::QuantumOpticsRepr)
-    @warn "Fock space cutoff is not specified so we default to 2"
-    return _ad₂
-end
-function express_nolookup(o::DestroyOp, r::QuantumOpticsRepr)
-    @warn "Fock space cutoff is not specified so we default to 2"
-    return _a₂
-end
-
+express_nolookup(s::FockState, r::QuantumOpticsRepr) = fockstate(FockBasis(r.fock_cutoff),s.idx)
+express_nolookup(s::CoherentState, r::QuantumOpticsRepr) = coherentstate(FockBasis(r.fock_cutoff),s.alpha)
+express_nolookup(o::NumberOp, r::QuantumOpticsRepr) = number(FockBasis(r.fock_cutoff))
+express_nolookup(o::CreateOp, r::QuantumOpticsRepr) = create(FockBasis(r.fock_cutoff))
+express_nolookup(o::DestroyOp, r::QuantumOpticsRepr) = destroy(FockBasis(r.fock_cutoff))
+express_nolookup(o::DisplaceOp, r::QuantumOpticsRepr) = displace(FockBasis(r.fock_cutoff), o.alpha)
 express_nolookup(x::MixedState, ::QuantumOpticsRepr) = identityoperator(basis(x))/length(basis(x)) # TODO there is probably a more efficient way to represent it
-function express_nolookup(x::IdentityOp, ::QuantumOpticsRepr)
-    b = basis(x)
-    if b!=inf_fock_basis
-        return identityoperator(basis(x)) # TODO there is probably a more efficient way to represent it
-    else
-        @warn "Fock space cutoff is not specified so we default to 2"
```

Comment:
there should still be a warning somewhere if the default cutoff is being used (maybe the warning can be moved to the constructor of `QuantumOpticsRepr`)